package.json

```diff
@@ -76,23 +76,68 @@
           "command": "azure-iot-edge.buildSolution",
           "group": "edge@1"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildSolution",
+          "group": "edge@1"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.buildAndPushSolution",
           "group": "edge@2"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildAndPushSolution",
+          "group": "edge@2"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.buildAndRunSolution",
           "group": "edge@3"
         },
+        {
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.buildAndRunSolution",
+          "group": "edge@3"
+        },
         {
           "when": "resourceFilename == deployment.template.json",
           "command": "azure-iot-edge.generateDeployment",
           "group": "edge@4"
         },
         {
-          "when": "resourceFilename == deployment.json",
+          "when": "resourceFilename == deployment.template.debug.json",
+          "command": "azure-iot-edge.generateDeployment",
+          "group": "edge@4"
+        },
+        {
+          "when": "resourceFilename == deployment.amd64.json",
+          "command": "azure-iot-edge.runSolution",
+          "group": "edge@0"
+        },
+        {
+          "when": "resourceFilename == deployment.amd64.debug.json",
+          "command": "azure-iot-edge.runSolution",
+          "group": "edge@0"
+        },
+        {
+          "when": "resourceFilename == deployment.arm32v7.json",
+          "command": "azure-iot-edge.runSolution",
+          "group": "edge@0"
+        },
+        {
+          "when": "resourceFilename == deployment.arm32v7.debug.json",
+          "command": "azure-iot-edge.runSolution",
+          "group": "edge@0"
+        },
+        {
+          "when": "resourceFilename == deployment.windows-amd64.json",
+          "command": "azure-iot-edge.runSolution",
+          "group": "edge@0"
+        },
+        {
+          "when": "resourceFilename == deployment.windows-amd64.debug.json",
           "command": "azure-iot-edge.runSolution",
           "group": "edge@0"
         },
```

Comment:
Reminder: update Toolkit to bind create deployment commands to paterns like deployment.amd64.json and deployment.amd64.debug.json